docs/management/managing-tags.asciidoc

```diff
@@ -0,0 +1,103 @@
+[role="xpack"]
+[[managing-tags]]
+== Tags
+
+The *tags* management section allows you to manage your tags, and assign them
+to existing saved objects.
+
+Assigning tags to objects helps to categorize them, and allows to easily find
+the objects by filtering by tag(s) in the <<managing-saved-objects, Saved Objects management section>>
+or from the other applications.
+
+To access the tags management page, open the main menu, then click on *Stack Management > Tags*.
+
+[role="screenshot"]
+image::images/tags/tag-management-section.png[Tags management section]
+
+[float]
+=== Required permissions
+
+The `Tag Management` {kib} privilege is required to access the *Tags* management section.
```

Comment:
It is a technical detail. Maybe we should provide it *after* the `Operations with tags` section to not overload a reader?

Reply:
You may be right. I followed the same order as the SO management documentation, but this specific doc/page seems outdated. @gchaps wdyt?